app/tda-project-ui/src/app/services/example/example.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ExampleService {
+  public VALUES_API_ENDPOINT = 'http://localhost:60443/api/values';
```

Comment:
I noticed that in `..Web.csproj`, the localhost was defined as `localhost:60443`, yet I've also noticed the frequent reference of `baseUrl = localhost:4200`. Is this intentional, having references/connections to both?

Reply:
`4200` is the port for the front-end, and `60443` is the port for the back-end.